consensus-types/epbs/indexed_payload_attestation_test.go

```diff
@@ -0,0 +1,114 @@
+package epbs
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/v5/consensus-types/primitives"
+	eth "github.com/prysmaticlabs/prysm/v5/proto/prysm/v1alpha1"
+	"github.com/prysmaticlabs/prysm/v5/testing/require"
+)
+
+func TestGetAttestingIndices(t *testing.T) {
+	testCases := []struct {
+		name             string
+		attestingIndices []primitives.ValidatorIndex
+	}{
+		{
+			name:             "Test 1",
+			attestingIndices: []primitives.ValidatorIndex{1, 2, 3},
+		},
+		{
+			name:             "Test 2",
+			attestingIndices: []primitives.ValidatorIndex{55,66,787},
+		},
+		{
+			name:             "Empty AttestingIndices",
+			attestingIndices: []primitives.ValidatorIndex{},
+		},
+		{
+			name:             "Nil AttestingIndices",
+			attestingIndices: []primitives.ValidatorIndex(nil),
+		},
+	}
+	for _, tt := range testCases {
+		t.Run(tt.name, func(t *testing.T) {
+			pa := &IndexedPayloadAttestation{
+				AttestingIndices: tt.attestingIndices,
+			}
+			got := pa.GetAttestingIndices()
+			require.DeepEqual(t, tt.attestingIndices, got)
+		})
+	}
+}
+
+func TestGetData(t *testing.T) {
+	testCases := []struct {
+		name string
+		data *eth.PayloadAttestationData
+	}{
+		{
+			name: "Test 1",
+			data: &eth.PayloadAttestationData{
+				Slot: primitives.Slot(1),
+			},
+		},
+		{
+			name: "Test 2",
+			data: &eth.PayloadAttestationData{
+				Slot: primitives.Slot(100),
+			},
+		},
+		{
+			name: "Zero slot",
+			data: &eth.PayloadAttestationData{
+				Slot: primitives.Slot(0),
+			},
+		},
+		{
+			name: "Nil AttestingIndices",
```

Comment:
```suggestion
			name: "Nil slot",
```
slot seems line up better with the other description.